helper/ssh-info.go

```diff
@@ -227,6 +230,34 @@ func sshInfoFromAPI(url *config.GitURL) (*SSHInfo, error) {
 	req.Header.Set("Accept", "application/json")
 
 	client := getHTTPClient()
+	gitConfig, err := goconfig.LoadAll("")
+	if err != nil {
+		return nil, err
+	}
+
+	if url.IsHTTPS() {
+		proxyRawURL = gitConfig.Get("https.proxy")
+		if proxyRawURL == "" {
+			proxyRawURL = os.Getenv("HTTPS_PROXY")
+		}
+	} else {
+		proxyRawURL = gitConfig.Get("http.proxy")
+		if proxyRawURL == "" {
+			proxyRawURL = os.Getenv("HTTP_PROXY")
+		}
+	}
+
+	if proxyRawURL != "" {
+		proxyURL, err = neturl.Parse(proxyRawURL)
+		if err != nil {
+			return nil, err
```

Comment:
Can we ignore this error and just show a warning message like:

    log.Warnf("fail to parse http proxy: %s", err)

Reply:
Yes, it's reasonable to just warn instead of stop the whole process when parse proxy url fails.

Reply:
Yes, it's reasonable to just warn instead of stop the whole process when parsing proxy url fails.

But it seems that log.Warnf() will print the warn message to file "actual" and fails to pass the test.  So for now I used log.Debugf() instead.